packages/nocodb/src/Noco.ts

```
@@ -100,6 +103,12 @@ export default class Noco {
     this.initCustomLogger(nestApp);
     nestApp.flushLogs();
 
+    try {
+      this.sharp = (await import('sharp')).default;
```

Biome

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)